### ISSUE TITLE
AllPay dependency causes errors when using certain economies.

### ISSUE THREAD

**Issue by lifehome** [Multiverse/Multiverse-Core#1592]
The outdated "Allpay" dependency is a big trouble on updating this plugin to 1.8-compatable, while I have forked @TWSSYesterday for further modification, I have realized that the "Allpay" and such things like GenericBanks are quite a pain in the ass to work with.

Any schedule on updating those dependency, in order to allow Multiverse-Core to "reborn"?

Note: If I didn't remove allpay, this happens >>  
https://gist.github.com/lifehome/a289e29900edd74269e3

**Comment by TWSSYesterday** [comment]
See https://github.com/FernFerret/AllPay
Ideally Vault would be a great thing to move to now it's more popular than it was when AllPay was used mainly.

**Comment by TWSSYesterday** [comment]
You may be interested in trying to bump https://github.com/FernFerret/AllPay/issues/26 if we choose not to move to vault. That being said, Vault is the best option for us.

**Comment by dumptruckman** [comment]
Why not just use Vault? MV supports it.

**Comment by lifehome** [comment]
I just have this done at lifehome/Multiverse-Core@270ac7a, perhaps one day needs to deprecate the use of AllPay?

**Comment by dumptruckman** [comment]
Oh right.. We already deprecated it. Why remove metrics though?

**Comment by lifehome** [comment]
@dumptruckman Uhm that's my fork, not really a patch tho... It's a dirty commit to get my server right back up. 

In addition, the use of Bukkit should be updated, such as pointing to SpigotMC. (just my flavour to get the latest 1.8 devs)

The `http://repo.bukkit.org/content/repositories/public` is already outdated as it stays on 1.7.9-R0.2

**Comment by dumptruckman** [comment]
So. What problem was leaving all pay in theregards causing?

**Comment by lifehome** [comment]
@TWSSYesterday got his point on FernFerret/AllPay#26.

**tl;dr** AllPay append somekindofweird prefix to `conomyBank6`, while Fe's not supported, iconomyBank6 turns to FeconomyBank6, causing error, blocking MVC running properly.

**Comment by lifehome** [comment]
Another thing is, while Vault is fully supported, AllPay shall only use for item, or Multiverse shall do its own way to deal with item economy, this would not cause the mess above happened.

i.e. No more catching up with API/econ-plugin updates

**Comment by dumptruckman** [comment]
But if you have Vault installed it should just take over and everything works, right?

**Comment by lifehome** [comment]
No, sadly it doesn't. If I use the jenkins build (686 or above), it gets error. No command, no execution, just a lot of error spamming.

**Comment by dumptruckman** [comment]
Ah, I see.

**Comment by dumptruckman** [comment]
This ought to fix the problem: 63d97c
